QUEST_LV_0100.tsv

```diff
@@ -1459,8 +1459,8 @@ QUEST_LV_0100_20150317_001454	The colleague who I used to work with has disappea
 QUEST_LV_0100_20150317_001455	Where is he. I am scared.
 QUEST_LV_0100_20150317_001456	{memo X}My colleague is dead?{nl}That's nonsense. If I had not tempted him..{nl}
 QUEST_LV_0100_20150317_001457	{memo X}This map is useless now since my colleague is dead.{nl}Please pray that my colleague would go to the Goddess safely.
-QUEST_LV_0100_20150317_001458	Damn. Monsters took my bag. {nl}The bag contained everything that is needed for my journey.. What should I do?{nl}
-QUEST_LV_0100_20150317_001459	The Goddess helped!{nl}Those monsters went down.
+QUEST_LV_0100_20150317_001458	Damn. Monsters took my bag. {nl}The bag contained everything I recorded on my journey.. What should I do?{nl}
```

Comment:
Damn -> I'm fucked (slang, but let's not do this :P) /I'm screwed/I'm doomed/I messed up.